src/assets/styles/styles/custom/_folding-section.scss

```diff
@@ -0,0 +1,30 @@
+.foldingSection {
+  height: 50px;
+
+  hr.middle {
+    width: 102%;
+    //   Fix, required to handle hr color correctly in electron
+    border: 0px;
+    border-top: 2px solid $table-border-color-grey;
+  }
```

Comment:
Nie wiem dlaczego teraz zaczął pojawiać się taki problem, ale linijki wyżej to naprawiają
![image](https://user-images.githubusercontent.com/21079319/103103415-34490d80-4621-11eb-84c6-18d9929755fb.png)